tendrl/common/manager/manager.py

```diff
@@ -0,0 +1,67 @@
+import abc
```

Comment:
scrap all the current gevent.greenlet.Greenlet threads and rename them to something below
- state sync (thread which syncs state from the system under management, eg: gluster cluster, ceph etc)
- job sync (thread to process jobs from queue)